grape-swagger.gemspec

```diff
@@ -15,8 +15,8 @@ Gem::Specification.new do |s|
   s.metadata['rubygems_mfa_required'] = 'true'
 
   s.required_ruby_version = '>= 3.0'
-  s.add_runtime_dependency 'grape', '>= 1.7', '< 3.0'
-  s.add_runtime_dependency 'rack-test', '~> 2'
+  s.add_dependency 'grape', '>= 2.2.0', '< 3.0'
```

Comment:
Not sure if it's OK to drop support for older grape versions - please let me know.

Otherwise we'd probably need to introduce dynamic behaviour at runtime based on the currently loaded grape version, something like

```ruby
if Gem::Version.new(Grape::VERSION) < '2.2.0'
  # old behaviour
else
  # new behaviour
end
```

Reply:
I would prefer we didn't unless we absolutely have to.